apps/web/components/apps/installation/AccountsStepCard.tsx

```diff
@@ -80,17 +87,18 @@ export const AccountsStepCard: FC<AccountStepCardProps> = ({ teams, personalAcco
           onClick={() => onSelect()}
           loading={loading}
         />
-        {teams.map((team) => (
-          <AccountSelector
-            key={team.id}
-            testId={`install-app-button-team${team.id}`}
-            alreadyInstalled={team.alreadyInstalled}
-            avatar={team.logo ?? ""}
-            name={team.name}
-            onClick={() => onSelect(team.id)}
-            loading={loading}
-          />
-        ))}
+        {!isConferencing &&
+          teams?.map((team) => (
```

Comment:
Should we instead have a prop `installableOnTeams` incase there are other apps we don't want to render teams on?